pkg/agent/agent_test.go

```diff
@@ -87,11 +87,11 @@ func (suite *AgentTestSuite) Testbootstrap() {
 
 func (suite *AgentTestSuite) TestSocketPermission() {
 	suite.agent = &Agent{
-		Catalog: suite.mockPluginCatalog,
-		CacheMgr:suite.mockCacheManager,
-		config:  suite.config}
+		Catalog:  suite.mockPluginCatalog,
+		CacheMgr: suite.mockCacheManager,
+		config:   suite.config}
```

Comment:
these are from `-s` or just plain `go fmt`?

Reply:
I believe `go fmt -s` implies `go fmt`

Reply:
Right - I guess my point was that it looks like we still have non-go-fmt'd files hanging around. I wonder if we should add a go fmt run to the travis tests